### PR TITLE
test review.....

### PR DESCRIPTION
**Pull request by fondadam**
test

### CODE REVIEW

**Comment by fondadam** [comment]
fixed #56

File: demo/doc/testReview.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# test
+
+```javascript
+const test = '123123';
+
+console.log('test.......')
+console.log('test2.......')
```
Comment (rock-zhang) [inline demo/doc/testReview.md]: bug

**Comment by fondadam** [comment]
test

File: demo/doc/testReview.md (hunk 1 above)
Comment (fondadam) [inline demo/doc/testReview.md]: bug fiex

File: demo/doc/testReview.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+console.log('test3.......')
+console.log('test4.......')
+console.log('test5.......')
+console.log('test6.......')
+```
+
+
+测试....
+
+测试222
```
Comment (fondadam) [inline demo/doc/testReview.md]: bug fixed

Comment (fondadam) [inline demo/doc/testReview.md]: is it right?

File: demo/doc/testReview.md (hunk 1 above)
Comment (fondadam) [inline demo/doc/testReview.md]: bug fixed

File: demo/doc/testReview.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+console.log('test.......')
+console.log('test2.......')
+console.log('test3.......')
+console.log('test4.......')
+console.log('test5.......')
+console.log('test6.......')
+```
+
+
```
Comment (fondadam) [inline demo/doc/testReview.md]: bug2

File: demo/doc/testReview.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+测试....
+
+测试222
+
+ceshi333
+ceshi333
+ceshi333
+ceshi4444
+ceshi4444
```
Comment (fondadam) [inline demo/doc/testReview.md]: test

Comment (fondadam) [inline demo/doc/testReview.md]: bug

File: demo/doc/testReview.md (hunk 1 above)
Comment (fondadam) [inline demo/doc/testReview.md]: bug fixed?

File: demo/doc/testReview.md (hunk 2 above)
Comment (fondadam) [inline demo/doc/testReview.md]: astart review 2222

File: demo/doc/testReview.md (hunk 1 above)
Comment (fondadam) [inline demo/doc/testReview.md]: a

File: demo/doc/testReview.md (hunk 2 above)
Comment (fondadam) [inline demo/doc/testReview.md]: 2

File: demo/doc/testReview.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+
+测试....
+
+测试222
+
+ceshi333
+ceshi333
+ceshi333
+ceshi4444
```
Comment (fondadam) [inline demo/doc/testReview.md]: 3

Comment (fondadam) [inline demo/doc/testReview.md]: aaaa

File: demo/index.js [hunk 6]
```diff
@@ -217,6 +217,7 @@ class Doc extends React.Component {
 
 class Standard extends React.Component {
     render() {
+        console.log('test.......');
```
Comment (fondadam) [inline demo/index.js]: sdasffasf

File: demo/doc/testReview.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+console.log('test2.......')
+console.log('test3.......')
+console.log('test4.......')
+console.log('test5.......')
+console.log('test6.......')
+```
+
+
+测试....
+
```
Comment (rock-zhang) [inline demo/doc/testReview.md]: Approve...

File: demo/doc/testReview.md (hunk 1 above)
Comment (farzer) [inline demo/doc/testReview.md]: 测试吧，single comment

File: demo/doc/testReview.md (hunk 7 above)
Comment (farzer) [inline demo/doc/testReview.md]: 测试，start review

File: demo/doc/testReview.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# test
```
Comment (GaaPill) [inline demo/doc/testReview.md]: hh😄

File: demo/doc/testReview.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+const test = '123123';
+
+console.log('test.......')
+console.log('test2.......')
+console.log('test3.......')
+console.log('test4.......')
+console.log('test5.......')
+console.log('test6.......')
+```
+
```
Comment (GaaPill) [inline demo/doc/testReview.md]: test comment

File: demo/doc/testReview.md (hunk 5 above)
Comment (farzer) [inline demo/doc/testReview.md]: add review comment

File: demo/doc/testReview.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+```javascript
+const test = '123123';
+
+console.log('test.......')
+console.log('test2.......')
+console.log('test3.......')
+console.log('test4.......')
+console.log('test5.......')
+console.log('test6.......')
```
Comment (rock-zhang) [inline demo/doc/testReview.md]: request changes

File: demo/doc/testReview.md (hunk 5 above)
Comment (GaaPill) [inline demo/doc/testReview.md]: test comment(≽◡≼)

File: demo/doc/testReview.md (hunk 8 above)
Comment (GaaPill) [inline demo/doc/testReview.md]: test

File: demo/doc/testReview.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+
+测试....
```
Comment (heyong-2015) [inline demo/doc/testReview.md]: start

File: demo/doc/testReview.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
+# test
+
+```javascript
```
Comment (GaaPill) [inline demo/doc/testReview.md]: test comment

File: demo/doc/testReview.md (hunk 11 above)
Comment (heyong-2015) [inline demo/doc/testReview.md]: bbb

File: demo/doc/testReview.md [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+const test = '123123';
+
+console.log('test.......')
+console.log('test2.......')
+console.log('test3.......')
+console.log('test4.......')
```
Comment (GaaPill) [inline demo/doc/testReview.md]: test comment ( ›◡ु‹ )

File: demo/doc/testReview.md (hunk 5 above)
Comment (fondadam) [inline demo/doc/testReview.md]: fied

File: demo/doc/testReview.md (hunk 13 above)
Comment (fondadam) [inline demo/doc/testReview.md]: flx

File: demo/doc/testReview.md (hunk 11 above)
Comment (fondadam) [inline demo/doc/testReview.md]: fox

Comment (fondadam) [inline demo/doc/testReview.md]: foxe

Comment (fondadam) [inline demo/doc/testReview.md]: foxe

File: demo/doc/testReview.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
+# test
```
Comment (heyong-2015) [inline demo/doc/testReview.md]: aaaa

File: demo/doc/testReview.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (heyong-2015) [inline demo/doc/testReview.md]: aaaa

**Pull request closed without merging**